src/ConfigProvider/PhpBackedConfigProvider.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tnapf\Config\ConfigProvider;
+
+use Tnapf\Config\Exceptions\InvalidConfigException;
+
+class PhpBackedConfigProvider implements ConfigProvider
+{
+    public function __construct(private readonly string $directory)
+    {
+    }
+
+    public function get(string $key): mixed
```

Comment:
Return type can be `?array`

Reply:
It doesn't have to be an array, it can be any value